lib/cisco_node_utils/vpc.rb

```diff
@@ -170,11 +168,13 @@ def fabricpath_multicast_load_balance
       config_get('vpc', 'fabricpath_multicast_load_balance')
     end
 
-    def fabricpath_multicast_load_balance=(val)
-      set_args_keys(state: val ? '' : 'no')
-      # This requires fabricpath_emulated_switch_id to be set first
-      fail 'fabricpath_switch_id configuration is required in the vPC domain' if
-        !fabricpath_emulated_switch_id && val
+    def fabricpath_multicast_load_balance=(state)
+      FabricpathGlobal.fabricpath_enable
+
+      fail 'This property also requires fabricpath_emulated_switch_id' if
```

Comment:
Note: This is prior code, I was just cleaning up the message.

In this case, if you configure `fabricpath multicast load-balance` first, without a `fabricpath switch-id` as well, it will configure just fine (no warning message) but it won't work until the switch-id is also added. 

IMO, this enters the too-helpful arena. We have plenty of prior art where independent commands are accepted and nvgen'd even though they have dependencies, so I suggest we remove this check altogether.